src/services/expense.service.js

```diff
@@ -0,0 +1,63 @@
+const { Expense } = require('./../models/Expense.model');
+const { Op } = require('sequelize');
+const normalize = ({ amount, category, id, note, spentAt, title, userId }) => {
+  return {
+    amount,
+    category,
+    id,
+    note,
+    spentAt,
+    title,
+    userId,
+  };
+};
+
+const getAll = async (filterParams = {}) => {
+  const { categories, from, to, ...query } = filterParams;
```

Comment:
This destructuring can be done in the function declaration:
```
const getAll = async ({ categories, from, to, ...query } = {}) => { ...
```
This will have the same effect